Module2_HM1/Module2_HM1/Result.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Module2_HM1
+{
+    internal class Result
+    {
+        public bool status { get; }
+        public string? errorMessage { get; }
+
+        public Result(bool status)
+        {
+            this.status = status;
+        }
+        public Result(bool status, string? errorMessage)
+        {
+            this.status = status;
```

Comment:
May be changed to public **Result(bool status, string? errorMessage): this(status)**